source/aggregation-pipeline-builder.txt

```diff
@@ -159,6 +159,25 @@ To view and change pipeline settings:
 
 .. _pipeline-custom-collation:
 
+Explain Plan for Pipelines
+--------------------------
+
+Use the :guilabel:`Explain Plan` to help you understand the performance of 
+your pipelines. You can view your :guilabel:`Explain Plan` at any point while 
```

Comment:
Perhaps "You can view the :guilabel:`Explain Plan` when 
creating or editing your pipeline".

Reply:
Done.